sites/main-site/src/content/blog/2024/tools-3_0_0.md

```diff
@@ -0,0 +1,299 @@
+In following releases we added subworkflows, switched to using nf-validation for schema validation and removed as much python code as possible from the template.
+While this happened the nf-core community grew to now 10,000 slack members including 2,000 active users, who are working on 113 pipelines (compared to 49 when 2.0 was released), 1343 modules and 72 subworkflows.
+With this growth we also saw new scientific areas being covered by nf-core pipelines, for example imaging data both on a [micro scale](https://nf-co.re/molkart/) and on a [global scale](https://nf-co.re/rangeland/).
+This and the adoption of the nf-core template and tooling even outside the nf-core community, convinced us to make the nf-core template even more flexible and to improve the user experience of the nf-core/tools.
+
+# What's new in nf-core/tools v3.0.0
+
+No worries, this is not as big of a change as the v2.0 release. The main things are:
+
+## ✨ New features
+
+- Enhanced pipeline template customisation
+  - The template has been divided into features that can be selectively included or excluded.
+    For example, you can now create a new pipeline without any traces of FastQC.
+    You can strip down the pipeline to the bare minimum and add only the tools you need.
+    For nf-core pipelines, certain core features (e.g., documentation, CI tests) remain mandatory, but you still have significant customisation flexibility.
+- New Text User Interface (TUI) for pipeline creation: A guided interface helps you through the process when running `nf-core pipelines create{:bash}` (don't worry - you can still use the CLI by providing all values as parameters).
+- [nf-schema](https://nextflow-io.github.io/nf-schema/latest/) has replaced nf-validation in the pipeline template
+- CI tests now use the nf-core tools version matching the pipeline's template version, reducing errors in PRs with each new tools release
+
+## ✨ New features
+
+- Enhanced pipeline template customisation
+  The template has been divided into features that can be selectively included or excluded.
+  For example, you can now create a new pipeline without any traces of FastQC.
+  You can strip down the pipeline to the bare minimum and add only the tools you need.
+  For nf-core pipelines, certain core features (e.g., documentation, CI tests) remain mandatory, but you still have significant customisation flexibility.
+- New Text User Interface (TUI) for pipeline creation
+  A guided interface helps you through the process when running `nf-core pipelines create{:bash}` (don't worry - you can still use the CLI by providing all values as parameters).
+- [nf-schema](https://nextflow-io.github.io/nf-schema/latest/) has replaced nf-validation in the pipeline template
+- CI tests now use the nf-core tools version matching the pipeline's template version
+  This will reduce errors in opened PRs with each new tools release
```

Comment:
@mashehu I think we both did the same change 😄

Reply:
oops, I removed one of them